R/toml_serialization.R

```diff
@@ -41,15 +41,17 @@ to_toml <- function(...,
   names <- names2(args)
 
   # We disallow unnamed top-level atomic arguments
-  invalid <- which(map_lgl(args, ~ is_atomic(.x) && !is.null(.x)))
+  invalid <- which(purrr::map2_lgl(names, args, ~ !nzchar(.x) && is.atomic(.y)))
+
   # If such args found, display an error message
   if (length(invalid) > 0) {
-    ui_throw(
-      get_toml_err_msg(),
+    cli::cli_abort(
       c(
-        make_idx_msg(invalid),
-        bullet_i("All top-level values should be named.")
-      )
+        get_toml_err_msg(),
+        "x" = make_idx_msg(invalid),
```

Comment:
This was a dirty hack before `{cli}` adoption. I think we can just use `x = "Unnamed arguments found at position(s): {invalid}."` and remove `make_idx_msg()`